packages/quiz/src/pages/api/answer.ts

```diff
@@ -21,7 +22,7 @@ const answer: NextApiHandler = async (
       const cookieHeader = req.headers.cookie as string
       const [subscriber, ckCookie] =
         await fetchConvertkitSubscriberFromServerCookie(cookieHeader)
-      const questionId = survey.id
+      const questionId = toLower(survey.id)
```

Comment:
Due to convertkit's transforming of field labels to lowercase when created, it broke when we used camelCase.

Reply:
Hope it was as easy to find as it was to fix! 🙌🏻